### DISCUSSION TITLE
2023/09/25/cloudflare-bulk-delete-dns-records/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#7] [Announcements]
# 2023/09/25/cloudflare-bulk-delete-dns-records/

域名从 Vercel 转到 Cloudlare 以后自动创建了一大堆记录，总共有一百多条，都是没用的解析。由于 Cloudflare 的控制台没有批量删除解析的功能，一条一条删很麻烦。不过我们可以通过 Cloudflare 的 API 功能来实现批量删除解析，准确地讲是全部删除，目前还不能实现删除指定的解析。

https://blog.iamsjy.com/2023/09/25/cloudflare-bulk-delete-dns-records/

**Comment by xloong** [comment]
不用整那么麻烦了 有个在线工具 [CFBatch.com](https://cfbatch.com) 可以直接在线批量删除域名DNS解析，免费额度足够使用了

**Reply by hmsjy2017** [reply]
![a26e7865c91fc41d3bf3f01bfdbdbd65](https://github.com/user-attachments/assets/398c2d82-f23b-4eca-89a1-cfb9bf379cce)
现在官方提供批量删除解析的选项了，连第三方工具都不需要

**Reply by xloong** [reply]
还真没注意多了这个多选框
